Extensions/anti_capitalism.js

```diff
@@ -19,6 +19,15 @@ XKit.extensions.anti_capitalism = new Object({
 			default: true,
 			value: true
 		},
+		"sidebar_ad": {
+			text: "Hide the Sidebar Ads",
+			default: true,
+			value: true
+		},
```

Comment:
wasn't hiding the sidebar ad a core part of the extension before?

Reply:
By core part I assume you mean not toggle-able? It was a setting (that didn't work on react); I just moved it up from the bottom of the list to the top and added a separator. The idea was to make it clear which settings currently have an effect on the react dash and which don't.